src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js

```diff
@@ -364,6 +374,31 @@ function ComposerWithSuggestions({
         [isKeyboardShown, isSmallScreenWidth, parentReportActions, report, reportActions, reportID, handleSendMessage, suggestionsRef, valueRef],
     );
 
+    const onChangeText = useCallback(
+        (commentValue) => {
+            if (syncSelectionWithOnChangeTextRef.current) {
+                setSelection({
+                    start: syncSelectionWithOnChangeTextRef.current.position,
+                    end: syncSelectionWithOnChangeTextRef.current.position,
+                });
```

Comment:
Right, this indeed seems to be dead, since the ref is set in call to `updateComment` - sorry for leaving that here, I finally moved `setSelection` inside `updateComment` and forgot about this one. I'll strip this off.

Reply:
Resolved in https://github.com/Expensify/App/pull/30835/commits/b2ff88cd3d44870fdf7677ebb5302df9568d2951